lib/matplotlib/__init__.py

```diff
@@ -126,6 +126,7 @@
                                 cycler)
 
 import numpy
+import numpy.ma
```

Comment:
Ensure that everyone can directly use `np.ma.<xyz>` without importing `numpy.ma`.